tests/Calculator/CalculatorTest.php

```diff
@@ -84,6 +84,10 @@ public function testRound()
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_EVEN));
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_ODD));
         $this->assertEquals('12', $calculator->round('12.50', Money::ROUND_HALF_DOWN));
+        $this->assertEquals('-2', $calculator->round(-1.5, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.578947368, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.5, Money::ROUND_HALF_UP));
```

Comment:
Fully agree: documentation is the keyword. And honestly, I also did not realize the difference. I think many people don't.

My vote would not go to `NORMAL`. I mean, what is normal? But I understand that my own suggestion is also not great. What about these?

`Money::ROUND_HALF_POSITIVE_INFINITY`
`Money::ROUND_HALF_NEGATIVE_INFINITY`
`Money::ROUND_HALF_UP`
`Money::ROUND_HALF_DOWN`

So I removed the word towards from the constants. The major question is: should we change the default behaviour? I can understand that the majority rather sees a negative number being rounded to positive infinity. You can even wonder when you need the default PHP behaviour at all.